src/Providers/LIAuthServiceProvider.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace LiveIntent\LaravelCommon\Providers;
+
+use Illuminate\Auth\RequestGuard;
+use Illuminate\Support\Facades\Auth;
+use LiveIntent\LaravelCommon\Auth\Guards\LITokenGuard;
+use Illuminate\Foundation\Support\Providers\AuthServiceProvider as ServiceProvider;
+
+class LIAuthServiceProvider extends ServiceProvider
+{
+    public function __construct($app)
+    {
+        parent::__construct($app);
+        $this->registerEnvironmentVariables();
+    }
+
+    /**
+     * Register any authentication / authorization services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        $this->registerCustomGuards();
+        $this->registerPolicies();
+    }
+
+    protected function registerEnvironmentVariables()
+    {
+        if (config('auth.li_token.keys.public') == null) {
+            config(['auth.li_token.keys.public' => env('TESSELLATE_PUBLIC_KEY') ]);
+        }
+
+        if (config('auth.guards.li_token') == null) {
+            config(['auth.guards.li_token' => [
+                'driver' => 'li_token',
+                'provider' => 'users',
+            ]]);
+        }
```

Comment:
I'm not sure i fully understand what is being done here. It looks like maybe you are trying to protect against a case where a consumer has a different config set up in their app maybe? Would appreciate clarification before I comment on it

Reply:
Same thing with this one -- this will automatically configure the `li_token` auth guard driver and provider if one is not specified already. If for whatever reason, a different provider is wanted, it can be overridden.

Another way we would support the "one function to enable auth" mechanism.

Another approach we could take here is to put this in the `liveintent.php` file and here we would load from that instead. Then we'd need all projects using laravel-common to update that file. This way it would be more consistent with expectations and less "magic". What do you think?